packages/react-components/react-toast/stories/Toast/ToasterLimit.stories.tsx

```diff
@@ -1,10 +1,11 @@
 import * as React from 'react';
 import { Toaster, useToastController, ToastTitle, Toast } from '@fluentui/react-toast';
-import { useId } from '@fluentui/react-components';
+import { useId, Button, Field, SpinButton } from '@fluentui/react-components';
 
 export const ToasterLimit = () => {
   const toasterId = useId('toaster');
   const { dispatchToast } = useToastController(toasterId);
+  const [limit, setLimit] = React.useState(3);
```

Comment:
Something does not work there.

- Click on a button infinite amount of times
- <img width="351" alt="image" src="https://github.com/microsoft/fluentui/assets/14183168/9ac95d5c-a5c0-4b9e-aea6-1c65dbfb702b">

Reply:
I cannot repro this actually, I have a limit 🙃

Reply:
It seems like this is a race condition of some sort, will look into it as a part of #28171